goty-server/src/main/kotlin/com/aleinin/goty/configuration/DefaultProperties.kt

```diff
@@ -1,32 +1,46 @@
 package com.aleinin.goty.configuration
 
+import com.aleinin.goty.properties.GotyQuestion
 import com.aleinin.goty.properties.Properties
 import org.springframework.boot.context.properties.ConfigurationProperties
 import org.springframework.format.annotation.DateTimeFormat
 import org.springframework.validation.annotation.Validated
 import java.time.ZonedDateTime
 import jakarta.validation.constraints.NotEmpty
 import jakarta.validation.constraints.NotNull
+import jakarta.validation.constraints.Positive
 import jakarta.validation.constraints.PositiveOrZero
+import java.time.ZoneId
 
 fun DefaultProperties.toProperties() = Properties(
-    gotyYear = this.gotyYear,
+    title = this.title,
+    goty = this.goty,
```

Comment:
What does this correspond to?

Reply:
```json
{
    "title": {
        "template": "The Midnight Watchmen's\nTop Games of the Year ${year}",
        "text": "The Midnight Watchmen's\nTop Games of the Year 2023"
    },
    "year": 2023,
    "goty": {
        "title": {
            "template": "Games of the Year",
            "text": "Games of the Year"
        },
        "question": {
            "template": "What are your favorite game(s) of ${year}?",
            "text": "What are your favorite game(s) of 2023?"
        },
        "rules": [
            {
                "template": "Voting closes ${deadline}.",
                "text": "Voting closes 01/01/2024 12:00 AM."
            },
            {
                "template": "May nominate any amount of games up to ${maxGames}.",
                "text": "May nominate any amount of games up to 10."
            },
            {
                "template": "Eligible games must have been released in ${year}.",
                "text": "Eligible games must have been released in 2023."
            },
            {
                "template": "Early access games are eligible if they were released in ${year}.",
                "text": "Early access games are eligible if they were released in 2023."
            },
            {
                "template": "DLC is not eligible.",
                "text": "DLC is not eligible."
            }
        ]
    },
    "tiePoints": [
        15,
        13,
        11,
        7,
        6,
        5,
        4,
        3,
        2,
        1
    ],
    "deadline": "2024-01-01T00:00:00-05:00",
    "hasGiveaway": true,
    "giveawayAmountUSD": 25,
    "defaultLocalTimeZone": "America/New_York"
}
```

![image](https://github.com/aleinin/game-of-the-year/assets/25621595/0feb4cf1-3c90-4edd-98eb-c7ca3738af2d)

Changed goty -> gotyQuestion